### ISSUE TITLE
[feature request] Write a jira comment directly from the interface

### ISSUE THREAD

**Issue by feuillemorte** [johannesjo/super-productivity#482]
At the moment I can change a jira status from the interface and there is a task ( #381 ) for creating a jira from the interface.
It would be good to add this feature for comments.

Current behavior: If you click on the button "write a comment" it will open a browser with this jira.

I would rename this button to smth like "Open in browser" and add a field like "notes" to be able to write a comment directly from the interface. Smth like this:

![image](https://user-images.githubusercontent.com/9434344/89529879-ca54d680-d7ed-11ea-90ab-51500fdfaf3a.png)

P.S. Unfortunately, I don't know angular and typescript, so, I can't really help with that. But I'd like to have more integration with jira like skipping a task from superproductivity if jira is in Done status and see the current jira status in the title of a task. I can create new tasks for this if it's make sense to implement.

Thank you!

**Comment by johannesjo** [comment]
Hey there! 

I agree that this might be nice to have a better way of doing this. The reason why I went with this approach is that Super Productivity probably never well be able to offer all the same capabilities as your Jira editor. Adding a comment is no problem, but something like @feuillemorte (ideally with autocomplete) or referencing issues is (A) hard to do and (B) likely to differ from the user experience on Jira not mainly but also because there are different versions of that as well. But maybe a simple comment field without all these features is enough? What do you think?

**Comment by feuillemorte** [comment]
For me it would be enough, because most of my comments are like "Blocked on/urls for PRs: <url>" or "Closing because of <...>". So, from my point of view this feature would be good for some "updating" comments. But for more complex comments/settings/smth else there is the `open in a browser` button. So, no need to repeat all jira functionality. 
If I could set the access level/view permissions (maybe globally?) to my comments (jira tasks on creating) that would more than enough!
But it's not very convenient for me to switch from superproductivity to jira, then I may write a comment there and I need to keep in mind that I need to finish his task in superproductivity because it can send time and change the status. In another case I have to go to superproductivity, finish a task and cancel the prompt about changing the status.
Maybe I'm using it in a wrong way, but it's a bit chaos for me switching between jira and superproductivity for simple actions.

```something like @feuillemorte (ideally with autocomplete) or referencing issues is (A) hard to do and (B) likely```
I'm not sure about it, but maybe if you post a link or a name than jira can show it properly on it's side.

So, I see the superproductivity as a place where you can do some simple actions in superproductivity directly (I mean notes, tasks, etc) and also in jira/gitlab/whatever and do not switch your focus to external applications if it's not needed to do some complex stuff. What do you think about it?

**Comment by johannesjo** [comment]
Thanks for explaining the use case to me. I am convinced! We should add adding basic comments ideally for all the issue providers. Currently, I am a bit busy with other features. So until I get to this PRs would be very welcome!

**Comment by johannesjo** [comment]
I am closing this in favor of #435
